charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,309 @@
+            action: delete
+
+      resource:
+        attributes:
+          - key: host.id
+            from_attribute: host.name
+            action: upsert
+          # TODO (chris): Upsert only when cluster name not found (resource detection override: true)
+          - key: k8s.cluster.name
+            action: upsert
+            value: {{ include "newrelic.common.cluster" . }}
+          - key: newrelicOnly
+            action: upsert
+            value: 'true'
+          - key: service.name
+            action: delete
+          - key: service_name
+            action: delete
+
+      k8sattributes:
+        auth_type: "serviceAccount"
+        passthrough: false
+        filter:
+          node_from_env_var: KUBE_NODE_NAME
+        extract:
+          metadata:
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.deployment.name
+            - k8s.namespace.name
+            - k8s.node.name
+            - k8s.pod.start_time
+        pod_association:
+          - sources: 
+            - from: resource_attribute
+              name: k8s.pod.uid
+
+      attributes/self:
+        actions:
+          - key: k8s.pod.name
+            action: update
+            from_attribute: pod
+          - key: k8s.deployment.name
+            action: update
+            from_attribute: deployment
+          - key: k8s.node.name
+            action: update
+            from_attribute: node
+          - key: k8s.namespace.name
+            action: upsert 
+            from_attribute: namespace
+
+      batch:
+
+    exporters:
+      otlphttp/newrelic:
+        endpoint: {{ .Values.newRelicEndpoint }}
+        headers:
+          api-key: ${env:NR_LICENSE_KEY}
+      # TODO(chris): uncomment when writing daemonset metrics through deployment
+      # otlphttp:
+      #   endpoint: "nr-k8s-otel-collector-gateway"
+      #   tls:
+      #     insecure: true
+      debug:
+        verbosity: detailed
+        sampling_initial: 5
+        sampling_thereafter: 200
+
+    service:
+      {{ if include "newrelic.common.verboseLog" . }}
+      telemetry:
+        logs:
+          level: "debug"
+      {{ end }}
+      pipelines:
+        metrics:
+          receivers: [hostmetrics, kubeletstats, prometheus]
+          processors:
+            # - transform/truncate
+            - filter/exclude_cpu_utilization
+            - filter/exclude_memory_utilization
+            - filter/exclude_memory_usage
+            - filter/exclude_filesystem_utilization
+            - filter/exclude_filesystem_usage
+            - filter/exclude_filesystem_inodes_usage
+            - filter/exclude_system_disk
+            - filter/exclude_system_paging
+            - filter/exclude_network
+            - attributes/exclude_system_paging
+            - resource
+            - k8sattributes
+            - attributes/self
+            - batch
+          exporters:
+            - otlphttp/newrelic
+            {{ if include "newrelic.common.verboseLog" . }}
```

Comment:
I am not sure if this would be "verbose" or even "trace". For example the agent does not print metrics in debug mode

Reply:
Removed the debug exporter :)